src/Codeception/Module/Db.php

```diff
@@ -395,6 +395,29 @@ protected function loadDumpUsingDriver()
      */
     public function haveInDatabase($table, array $data)
     {
+        $lastInsertId = $this->insertInDatabase($table, $data);
+
+        $this->addInsertedRow($table, $data, $lastInsertId);
+
+        return $lastInsertId;
+    }
+
+    /**
+     * Inserts an SQL record into a database.
```

Comment:
how is this different from `haveInDatabase`?
I think having both methods are confusing